web/styles/app_components.css

```diff
@@ -112,7 +112,7 @@ input::placeholder {
 /* -- base button styling -- */
 
 .button {
-    padding: 7px 14px;
+    padding: 8px 15px;
```

Comment:
With the loss of a `border`, we bump the padding by a pixel in all dimensions to maintain the current button size. The same adjustment is present below on `.small`.